parcels/particle.py

```diff
@@ -457,6 +473,94 @@ def show(self, **kwargs):
         plt.show()
         plt.pause(0.0001)
 
+    def show_velocity(self, **kwargs):
+        # time at which to plot velocity
+        t = kwargs.get('t', None)
+        if t is None:
+            t = self.particles[0].time
+        # flag to drawing land on plot
+        land = kwargs.get('land', False)
+        # plot domain latitude longitude parameters
+        latN = kwargs.get('latN', np.nan)
+        latS = kwargs.get('latS', np.nan)
+        lonE = kwargs.get('lonE', np.nan)
+        lonW = kwargs.get('lonW', np.nan)
+        # maximum speed for vector coloring
+        vmax = kwargs.get('vmax', None)
+        # filename to saving to
+        savefile = kwargs.get('savefile', None)
+
+        if isinstance(t, datetime):
+            t = (t - self.grid.U.time_origin).total_seconds()
+        if isinstance(t, delta):
+            t = t.total_seconds()
+
+        if not np.isnan(latN):
+            latN = nearest_index(self.grid.U.lat, latN)
+            latS = nearest_index(self.grid.U.lat, latS)
+            lonE = nearest_index(self.grid.U.lon, lonE)
+            lonW = nearest_index(self.grid.U.lon, lonW)
+            lon = self.grid.U.lon[lonW:lonE]
+            lat = self.grid.U.lat[latS:latN]
+        else:
+            lon = self.grid.U.lon
+            lat = self.grid.U.lat
+
+        # time interpolation of velocity field
+        idx = self.grid.U.time_index(t)
+        U = np.array(self.grid.U.temporal_interpolate_fullfield(idx, t))
+        V = np.array(self.grid.V.temporal_interpolate_fullfield(idx, t))
+        if not np.isnan(latN):
+            U = U[latS:latN, lonW:lonE]
+            V = V[latS:latN, lonW:lonE]
+
+        # configuring plot
+        lat_median = np.median(lat)
+        lon_median = np.median(lon)
+        plt.figure()
+        m = Basemap(projection='merc', lat_0=lat_median, lon_0=lon_median,
+                    resolution='h', area_thresh=100,
+                    llcrnrlon=lon[0], llcrnrlat=lat[0],
+                    urcrnrlon=lon[-1], urcrnrlat=lat[-1])
```

Comment:
Have you checked that this doesn't crash when `Basemap` is not available? I.e. what happens when there is an exception up in line 16 of this file?